applications/crossbar/doc/agents.md

```diff
@@ -0,0 +1,44 @@
+
+#API for managing agent status
+This API provides convenient way for agent status management without need to dial feature codes. It's useful for various call center agent/supervisor interfaces.
+
+### Log In/Log Out agent to/from some queue
+
+/agents/AID/logged_in (GET, POST):
```

Comment:
I am not a fan of this layout.

URIs should be nouns, things, objects; "logged_in" isn't that. I'd prefer something like:

`/agents/{AGENT_ID}/status`
* GET: Fetches status on each queue an agent is a member of
* GET `?queue_id={QUEUE_ID}`: fetch status for queue `{QUEUE_ID}`
* POST `{"data":{"action":{ACTION}}}`: performs `{ACTION}` on all queues of the agent
* POST `{"data":{"action":{ACTION}, "queue_id":{QUEUE_ID}}}`: performs `{ACTION}` on queue `{QUEUE_ID}`

Thoughts?